caproto/_state.py

```diff
@@ -180,6 +181,7 @@
             ErrorResponse: MUST_CLOSE,
         },
         CLOSED: {
+            ClearChannelResponse: CLOSED,
```

Comment:
This is probably not the right fix anyway,  there seems to be a race condition when things are torn down where the channel is closed twice.